content/en/docs/tasks/administer-cluster/kubelet-config-file.md

```diff
@@ -115,11 +115,9 @@ The suffix of a valid kubelet drop-in configuration file **must** be `.conf`. Fo
 The kubelet processes files in its config drop-in directory by sorting the **entire file name** alphanumerically.
 For instance, `00-kubelet.conf` is processed first, and then overridden with a file named `01-kubelet.conf`.
 
-These files may contain partial configurations and might not be valid config files by themselves.
-Validation is only performed on the final resulting configuration structure
-stored internally in the kubelet.
-This offers you flexibility in how you manage and combine kubelet configuration that comes from different sources.
-However, it's important to note that the behavior varies based on the data type of the configuration fields.
+These files should contain complete configurations to avoid failure due to invalid configuration files. 
```

Comment:
So the partial configuration doesn't work?

Reply:
> So the partial configuration doesn't work?

Yeah, the partial configuration should work. Let me update that information.